tests/unit/via/views/view_video_test.py

```diff
@@ -20,6 +20,9 @@ def test_it(self, pyramid_request, Configuration, youtube_service, video_url):
         response = youtube(pyramid_request)
 
         youtube_service.get_video_id.assert_called_once_with(video_url)
+        Configuration.extract_from_params.assert_called_once_with(
+            {"via.foo": "foo", "via.bar": "bar"}
+        )
```

Comment:
Added a missing assertion. This assertion might have caught the mistake when my change that broke this changed this line to pass `{}` to `extract_from_params()` instead of the actual params